examples/basic/metro.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
why is this needed?

Reply:
this is part of the RN 0.60 template. I'm hoping to simplify some of this in the future but was trying to do the minimum possible to get the example running.